util.h

```diff
@@ -70,7 +70,7 @@ typedef U64TYPE perl_drand48_t;
 #else
 
 struct PERL_DRAND48_T {
-    U16 seed[3];
+    U16     seed[3];
```

Comment:
Reformat structs just applies the same layout logic to all structs.  Its not just lining them up, its also deciding which tab stop they are lined up on, and I believe I deliberately did this to ensure there was at least one empty tab stop in between type and field. I can undo it, but i found it made things more legible with structs with 3 character types, which is fairly common for us, also it provides a bit more leeway to add a longer type without needing to reindent the struct again.  

If there was consensus we should do something else I am fine with that, but i made this choice because it opened up the struct definitions and made them "breathe" a bit which I think makes them more readable and comprehensible. I guess we could say "less than K lines uses a minimal indent", with K being 3 or something, and for structs with more members we would use the current logic. We have some structs that have dozens of integer type fields, and i found that when they were "close packed" (eg allow a minimum of one space between type and field) they became an eyesore to read.

In short, while I agree this might be controversial, this is "not a bug". :-)

I can try re-pushing without this deliberate rule and we can take a look, but it will affect every struct. Which is a point worth considering with these changes in the PR, the logic might make a few minor cases slightly "worse" in some aesthetic sense while fixing a whole bunch of other cases and being a net win.  That is a the problem with standardizing things, it can be hard to codify rules that lead to the desired outcome in every case across all our code.  

Also I guess at the end we can just rebase-interactive some of the controversial parts, but I'm hoping to automate the vast bulk of this so that if we were to apply it, and someone had changes that turned into conflict hell, we could run the script over their pre-image headers to get the same format, which then could be merged with far less conflicts.